mariza/styles.css

```
@@ -0,0 +1,176 @@
+* {
+  box-sizing: border-box;
+}
+.header {
+  background-color: #ff8c00;
+  display: inline-block;
+  color: #ffffff;
+  font-family: "Open Sans", sans-serif;
+  width: 100%;
+}
+h1 {
+  background-color: #ff8c00;
+  font-size: 30px;
+  font-style: regular;
+  font-weight: medium;
+  font-family: "Raleway", sans-serif;
+  color: white;
+}
+.hours {
+  display: inline-block;
+  background-color: #ff8c00;
+  font-size: 12px;
+  color: white;
+  font-weight: light;
+  font-family: "Open Sans", sans-serif;
+}
+.times {
+  display: block;
+}
+.appointment {
+  font-style: italic;
+}
+.interior {
+  width: auto;
+}
+.interior-style {
+  object-fit: cover;
+  object-position: 75% 35%;
+  height: 300px;
+  width: 100%;
+  margin-bottom: -20px;
+}
+h2 {
+  font-size: 30px;
+  font-weight: black;
+  font-family: "Raleway", sans-serif;
+}
+h3 {
+  font-size: 16px;
+  font-style: regular;
+  font-weight: bold;
+  font-family: "Open Sans", sans-serif;
+  text-transform: uppercase;
+}
+h4 {
+  font-size: 18px;
+  font-weight: 300;
+  font-style: regular;
+  font-family: "Raleway", sans-serif;
+  color: black;
+  margin-top: -5px;
+}
+h5 {
+  font-size: 16px;
+  font-weight: normal;
+  font-family: "Oswald", serif;
+}
+li {
+  text-transform: capitalize;
+  display: block;
```

`li` elements are block elements by default, so this can be removed:
```suggestion
```
if the effect you're trying to get is losing the bullets next to each `li`, you want to do that via
```suggestion
  list-style: none;
```